src/python/oftest/dataplane.py

```diff
@@ -81,7 +81,10 @@ def recv(self):
         Receive a packet from this port.
         @retval (packet data, timestamp)
         """
-        pkt = afpacket.recv(self.socket, self.RCV_SIZE_DEFAULT)
+        try:
+            pkt = afpacket.recv(self.socket, self.RCV_SIZE_DEFAULT)
+        except:
+            return (None, None)
```

Comment:
instead of returning (None, None) in case of any exception, how about letting the exception propagate upwards, and then catching the specific exception in the caller?  i.e. put the try/except around the call to port.recv (line 212 in the original file)?